src/core/packages/saved-objects/migration-server-internal/src/zdt/utils/generate_additive_mapping_diff.test.ts

```diff
@@ -78,7 +77,6 @@ describe('generateAdditiveMappingDiff', () => {
 
     expect(addedMappings).toEqual({
       foo: foo.mappings,
-      bar: bar.mappings,
```

Comment:
Types that do not define modelVersions (see definition of _bar_ above) are now defaulted to `10.0.0` as minimum virtual version.

Thus, any older version `migrations:` will be considered "already performed". This is safe to assume thanks to the _Kibana version check_ also introduced in this PR. This is why both changes need to go together.